fastlane/screenshots/README.md

```diff
@@ -0,0 +1,129 @@
+# How to generate & update Play Store Screenshots
+
+The screenshots used for the Play Store come in multiple device sizes (phone vs tablet) and many locales (currently 16 different languages), so it would be unreasonable to expect designers to generate all the variants by hand every time we want to update those screenshots. For this reason, the generation of those Play Store screenshots are automated with fastlane.
+
+## High Level overview
+
+Generating new screenshots for the Play Store consists of following this multi-stage process:
+
+ - First, we have UI tests in the codebase — namely [`WPScreenshotTest`](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java) and [`JPScreenshotTest`](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/JPScreenshotTest.java) — which are responsible for launching the app and navigating through the various screens we need screenshot for, and taking a "raw screenshot" (screen capture) of those screens.
+    - If you plan to update the Play Store screenshots to include different screens that the ones currently included, or if the navigation of the app to get to those screens changed as the app evolved, you'll need to update those `WPScreenshotTest` and `JPScreenshotTest` test suites accordingly.
```

Comment:
I suggest removing "as the app evolved" to get to the point of the item (update the tests) faster.

```suggestion
    - If you plan to update the Play Store screenshots to include different screens that the ones currently included, or if the navigation of the app to get to those screens changed, you'll need to update those `WPScreenshotTest` and `JPScreenshotTest` test suites accordingly.
```